src/Sylius/Bundle/ApiBundle/StateProvider/Admin/Account/ResetPassword/ItemProvider.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\StateProvider\Admin\Account\ResetPassword;
```

Comment:
```suggestion
namespace Sylius\Bundle\ApiBundle\StateProvider\Admin\ResetPassword;
```
or
```suggestion
namespace Sylius\Bundle\ApiBundle\StateProvider\Admin\AdminUser\ResetPassword;
```
but as the command is in the `Account` directory, the current namespace has probably more sense 🤔

Reply:
👍 in favour keeping namespace as flat as possible:

```
namespace Sylius\Bundle\ApiBundle\StateProvider\Admin\ResetPassword;
```